src/main.rs

```diff
@@ -127,6 +129,16 @@ fn bash(imports_path: Option<&&str>) {
     print!("{}", TABRY_BASH_SH);
 }
 
+const TABRY_FISH_SH: &str = include_str!("../shell/tabry_fish.fish");
+fn fish(imports_path: Option<&&str>) {
+    if let Some(path) = imports_path {
+        println!("set -x _tabry_rs_imports_path '{}'", escape(path));
+    }
+
+    println!("set -x _tabry_rs_executable '{}'", escaped_exe());
```

Comment:
Since this is exported, does it make since to name it uppercase?
Also, could you please name it either tabry_executable/TABRY_EXECUTABLE, or tabry_fish_executable/TABRY_FISH_EXECUTABLE, for consistency? (I think I'm going to name most things just 'tabry', I may in the future change the repo name)